### DISCUSSION TITLE
Your Thoughts on Badges?

### DISCUSSION THREAD

**Discussion by zerox-alpha** [Schweinepriester/github-profile-achievements#940] [Ideas]
Hey everyone,

I'm keen on sprucing up our project's GitHub with badges. What badges do you find useful or interesting? Any ideas on how we can earn them? Drop your thoughts below!

**Comment by jclizarraga** [comment]
Answered  people questions.

**Comment by 23Edu4rd0** [comment]
It might seem useless at first, but badges show that you're actively involved—committing, working on projects, participating in discussions, etc. It's nice to see your progress being recognized and rewarded.
Just make pull requests and answered people and you will get it

**Comment by amirmohideen** [comment]
i answered 2 and it was marked but didnt get the badge. I am gonna try a 3rd one lets see

**Comment by mansi-nirwal** [comment]
> Hey everyone,
> 
> I'm keen on sprucing up our project's GitHub with badges. What badges do you find useful or interesting? Any ideas on how we can earn them? Drop your thoughts below!
